docs/en/connector-v2/sink/Doris.md

```diff
@@ -17,67 +30,130 @@ Version Supported
+| sink.buffer-count   | int    | No       | 3          | the buffer count to cache data for stream load.                                                                                                                                                                                                                                              |
+| doris.config        | map    | yes      | -          | This option is used to support operations such as `insert`, `delete`, and `update` when automatically generate sql,and supported formats.                                                                                                                                                    |
+
+## Data Type Mapping
+
+| Doris Data type | SeaTunnel Data type                     |
+|-----------------|-----------------------------------------|
+| BOOLEAN         | BOOLEAN                                 |
+| TINYINT         | TINYINT                                 |
+| SMALLINT        | SMALLINT<br/>TINYINT                    |
+| INT             | INT<br/>SMALLINT<br/>TINYINT            |
+| BIGINT          | BIGINT<br/>INT<br/>SMALLINT<br/>TINYINT |
+| LARGEINT        | BIGINT<br/>INT<br/>SMALLINT<br/>TINYINT |
+| FLOAT           | FLOAT                                   |
+| DOUBLE          | DOUBLE<br/>FLOAT                        |
+| DECIMAL         | DECIMAL<br/>DOUBLE<br/>FLOAT            |
+| DATE            | DATE                                    |
+| DATETIME        | TIMESTAMP                               |
+| CHAR            | STRING                                  |
+| VARCHAR         | STRING                                  |
+| STRING          | STRING                                  |
+| ARRAY           | ARRAY                                   |
+| MAP             | MAP                                     |
+| JSON            | STRING                                  |
+| HLL             | Not supported yet                       |
+| BITMAP          | Not supported yet                       |
+| QUANTILE_STATE  | Not supported yet                       |
+| STRUCT          | Not supported yet                       |
 
-### password [string]
 
-`Doris` user password
-
-### table.identifier [string]
-
-The name of `Doris` table
-
-### sink.label-prefix [string]
-
-The label prefix used by stream load imports. In the 2pc scenario, global uniqueness is required to ensure the EOS semantics of SeaTunnel.
-
-### sink.enable-2pc [bool]
-
-Whether to enable two-phase commit (2pc), the default is true, to ensure Exactly-Once semantics. For two-phase commit, please refer to [here](https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD).
-
-### sink.enable-delete [bool]
+#### Supported import data formats
```

Comment:
`Supported Import Data Formats`?

Reply:
This is the original category and can be deleted if not needed